<launch>
	<node pkg="zeabus_elec_ros_peripheral_bridge" type="zeabus_elec_ros_peripheral_bridge" name="Zeabus_Elec_Peripheral_bridge" output="screen">
		<param name="comm1baudrate" value="115200"/>
		<param name="comm2baudrate" value="115200"/>		
                <param name="IODirectionA" value="65535"/>
                <param name="IOStateA" value="0"/>
                <param name="IODirectionB" value="65535"/>
                <param name="IOStateB" value="0"/>
	</node>
</launch>
